package.json

```diff
@@ -4,6 +4,7 @@
   "private": false,
   "dependencies": {
     "bulma": "^0.8.0",
+    "date-fns": "^2.11.0",
```

Comment:
Haven't heard about this lib before. Does it have any benefits in front of [moment.js](https://momentjs.com/)

Reply:
it's functional, rather than OOP like moment, so you can just import the functions you need. It's faster also